src/app/shell/admin-tools/data/history-log/history-log-filters/history-log-filters.component.html

```diff
@@ -24,6 +22,11 @@
         <mat-option *ngFor="let option of dropdownOptions" [value]="option.value" class="dropdown-option">{{ option.label }}</mat-option>
       </mat-select>
     </mat-form-field>
+  </div>
+  <div class="filters-actions">
+    <div class="reset" (click)="onResetFilters()">
```

Comment:
![image](https://user-images.githubusercontent.com/27493959/208448856-1a245443-6b97-4a76-9d5f-238f397b5c60.png)
text is missing